app/graphql/types/project_type.rb

```diff
@@ -24,6 +24,8 @@ class ProjectType < Types::BaseObject
 
     field :enabled_digital_object_types, [String], null: false
 
+    field :has_asset_rights, Boolean, null: true
```

Comment:
Given my lack of experience with GraphQL, this may be completely off-base, but in my code I had 'null: false' because:
1) Mimics column definition in db.
2) Set to false for similar fields, such as is_primary.

Reply:
I didn't see the default, let me fix this and push the change